goodtablesio/models/task.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import datetime
+from sqlalchemy.orm import relationship
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy import Column, Unicode, DateTime,  ForeignKey
+from goodtablesio.models.base import Base, BaseModelMixin, make_uuid
+log = logging.getLogger(__name__)
+
+
+class Task(Base, BaseModelMixin):
```

Comment:
So I've named it `Task` intentionally because it's literally celery task wrapper (with exact task_id etc). So there is rationale - https://github.com/frictionlessdata/goodtables.io/pull/174#issuecomment-286385767 (about keep it simple).

But I suppose you're right about still preserving room for having composite `internal jobs`. So going to change it to `InternalJob` (in `tasks.base`).